docs/installation/firewall.rst

```diff
@@ -43,7 +43,7 @@ In return, Gemnasium syncer provides all the metadata corresponding to this requ
 
 .. note:: Private dependencies are completely ignored by the syncer.
 
-Advisories can be created at any time on Gemnasium.com, that's why your instance synchronizes every five minutes.
-If one of your project is using a dependency affected by a security issue, you will be instantly notified by your instance.
+Advisories can be created at any time on Gemnasium.com; that's why your instance synchronizes every five minutes.
```

Comment:
I think this is not the case anymore, it's hourly now (I know it's not one of your change, but let's keep the doc clean :)